digitalocean/resource_digitalocean_project_test.go

```diff
@@ -43,6 +44,75 @@ func TestAccDigitalOceanProject_CreateWithDefaults(t *testing.T) {
 	})
 }
 
+func TestAccDigitalOceanProject_CreateWithIsDefault(t *testing.T) {
+	expectedName := generateProjectName()
+	expectedIsDefault := "true"
+	createConfig := fixtureCreateWithIsDefault(expectedName, expectedIsDefault)
+
+	var (
+		originalDefaultProject = &godo.Project{}
+		client                 = &godo.Client{}
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+
+			// Get an store original default project ID
+			client = testAccProvider.Meta().(*CombinedConfig).godoClient()
+			defaultProject, _, defaultProjErr := client.Projects.GetDefault(context.Background())
+			if defaultProjErr != nil {
+				t.Errorf("Error locating default project %s", defaultProjErr)
+			}
+			originalDefaultProject = defaultProject
+		},
+		ProviderFactories: testAccProviderFactories,
+		CheckDestroy:      testAccCheckDigitalOceanProjectDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config:             createConfig,
+				ExpectNonEmptyPlan: true,
```

Comment:
Without this, I was getting the following error:

```
resource_digitalocean_project_test.go:57: Step 1/1 error: After applying this test step and performing a `terraform refresh`, the plan was not empty.
        stdout
        
        
        An execution plan has been generated and is shown below.
        Resource actions are indicated with the following symbols:
          ~ update in-place
        
        Terraform will perform the following actions:
        
          # digitalocean_project.myproj will be updated in-place
          ~ resource "digitalocean_project" "myproj" {
                id         = "1e7c1f73-9924-4cb2-b706-1ef6bf339dbb"
              ~ is_default = false -> true
                name       = "tf-proj-test-4023944325181221421"
                # (6 unchanged attributes hidden)
            }
        
        Plan: 0 to add, 1 to change, 0 to destroy.
```

ExpectNonEmptyPlan can be set to true for specific types of tests that are looking to verify that a diff occurs. Doc [here](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource#TestStep)